src/esp/geo/geo.cpp

```diff
@@ -2,6 +2,9 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
+#include <Corrade/Containers/ArrayViewStl.h>
+#include <Magnum/Math/FunctionsBatch.h>
+
```

Comment:
What are these additions for? They don't seem to be needed.

Reply:
This is I think related to the disabled `BUILD_DEPRECATED` flag -- I got rid of some inter-header dependencies for faster compile times but kept the includes for backwards compatibility when the flag is disabled.